units/src/amount.rs

```diff
@@ -98,8 +98,8 @@ impl Denomination {
 
 /// These form are ambigous and could have many meanings.  For example, M could denote Mega or Milli.
 /// If any of these forms are used, an error type PossiblyConfusingDenomination is returned.
-const CONFUSING_FORMS: [&str; 9] =
-    ["Msat", "Msats", "MSAT", "MSATS", "MSat", "MSats", "MBTC", "Mbtc", "PBTC"];
+const CONFUSING_FORMS: [&str; 6] =
+    ["MBTC", "Mbtc", "CBTC", "Cbtc", "UBTC", "Ubtc"];
```

Comment:
Strictly speaking C and U are not confusing but I agree with banning them because they look too weird. :)